app/services/foreman_ansible/proxy_selector.rb

```diff
@@ -0,0 +1,21 @@
+module ForemanAnsible
+  # Contains proxy selection rules for a host playbook run
+  class ProxySelector < ::ForemanTasks::ProxySelector
+    # rubocop:disable Metrics/MethodLength
+    def available_proxies(host)
+      proxies = {}
+
+      proxies[:fallback] = host.smart_proxies.with_features('Ansible')
+      proxy_scope = if Taxonomy.enabled_taxonomies.any?
+                      ::SmartProxy.with_taxonomy_scope_override(host.location,
```

Comment:
Nevermind, it does work after reverting https://github.com/theforeman/foreman/commit/35ed04f95f14c934ffebc874e90a3a4a6684fe71